Registration/RegisterWithAzure.psm1

```diff
@@ -346,7 +346,7 @@ function Set-AzsRegistration{
         [switch] $MarketplaceSyndicationEnabled = $true,
```

Comment:
Do you need a similar PR for AzureRM version?

Please make sure that the cahgnes are backward compatible given that we dont version this script

Reply:
PR for AzureRM --- https://github.com/Azure/AzureStack-Tools/pull/619